examples/consensus_pub_sub/main.go

```diff
@@ -14,14 +14,11 @@ func main() {
 	var client *hedera.Client
 	var err error
 
-	if os.Getenv("HEDERA_NETWORK") == "previewnet" {
-		client = hedera.ClientForPreviewnet()
-	} else {
-		client, err = hedera.ClientFromConfigFile(os.Getenv("CONFIG_FILE"))
-
-		if err != nil {
-			client = hedera.ClientForTestnet()
-		}
+	net := os.Getenv("HEDERA_NETWORK")
+	client, err = hedera.ClientForName(net)
```

Comment:
Could simplify this a bit to `client, err := hedera.ClientForName(os.Getenv("HEDERA_NETWORK"))`